src/persistence/person-rollenzuweisung.entity.ts

```diff
@@ -1,17 +1,21 @@
 import { TimestampedEntity } from './timestamped.entity.js';
-import { Entity, ManyToOne } from '@mikro-orm/core';
-import { PersonEntity } from '../modules/person/persistence/person.entity.js';
+import { Entity, ManyToOne, Property } from '@mikro-orm/core';
 import { RolleEntity } from './rolle.entity.js';
-import { SchulstrukturknotenEntity } from './schulstrukturknoten.entity.js';
 
 @Entity({ tableName: 'person_rollenzuweisung' })
 export class PersonRollenzuweisungEntity extends TimestampedEntity<PersonRollenzuweisungEntity, 'id'> {
-    @ManyToOne()
-    public person!: PersonEntity;
+    /**
+     * Links to Person
+     */
+    @Property()
+    public person!: string;
 
     @ManyToOne()
     public role!: RolleEntity;
 
-    @ManyToOne()
-    public schoolStructureNode!: SchulstrukturknotenEntity;
+    /**
+     * Points to Schulstrukturknoten
+     */
+    @Property()
+    public schoolStructureNode!: string;
```

Comment:
Don't we have the convention now, to name it in German "schulstrukturknoten"
```suggestion
    public schulstrukturknoten!: string;
```